core/templates/services/rte-helper.service.spec.ts

```diff
@@ -330,7 +330,6 @@ describe('Rte Helper Service', () => {
 
   it('should evalute when rich text component is not inline', () => {
     expect(rteHelperService.isInlineComponent('video')).toBe(false);
-    expect(rteHelperService.isInlineComponent('tabs')).toBe(false);
```

Comment:
Why drop this line?

Reply:
I did it because on the other PR it was done but after looking into it I think it was erased by mistake since I can't see a reason to why it should be erased.